pinner/pinmgr.go

```diff
@@ -143,7 +141,7 @@ func newPinManager(pinfunc PinFunc, scf PinStatusFunc, opts *PinManagerOpts, log
 	if opts.QueueDataDir == "" {
 		log.Fatal("Deque needs queue data dir")
 	}
-
+	duplicateGuard := buildDupliceGuardFromPinQueue(opts.QueueDataDir, log)
```

Comment:
this should say duplicate, not duplice, unless there's some other meaning @snissn

Reply:
lol no just bad @ spell

Reply:
https://github.com/application-research/estuary/pull/945/commits/7d1e3147a4d77ec667641511bde2b23837f29daa

Reply:
fixed